### ISSUE TITLE
Skin FIP <Img...> tag missing closing .../>

### ISSUE THREAD

**Issue by Phanoix** [gctools-outilsgc/gcconnex#8]
SkinFIP <Img...> tag missing closing .../>
==>Add the closing '/' to the image tag.
